src/components/AnonymousChat/ChatDetailHeader.tsx

```diff
@@ -21,10 +21,10 @@ const styles = StyleSheet.create({
     alignItems: 'center'
   },
   bgsigned_primary: {
-    backgroundColor: COLORS.signed_primary
+    backgroundColor: COLORS.signed_secondary
   },
   bgBondiBlue: {
-    backgroundColor: COLORS.anon_primary
+    backgroundColor: COLORS.anon_secondary
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color scheme for `bgsigned_primary` and `bgBondiBlue` has been changed. Ensure that these changes are consistent with the overall design and accessibility standards, such as contrast ratio for readability.
```diff
-    backgroundColor: COLORS.signed_primary
+    backgroundColor: COLORS.signed_secondary

-    backgroundColor: COLORS.anon_primary
+    backgroundColor: COLORS.anon_secondary
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->